base/mathconstants.jl

```diff
@@ -85,6 +85,7 @@ catalan
 for T in (Irrational, Rational, Integer, Number)
     Base.:^(::Irrational{:ℯ}, x::T) = exp(x)
 end
+@generated Base.literal_pow(::typeof(^), ::Irrational{:ℯ}, ::Val{p}) where {p} = exp(p)
```

Comment:
this doesn't need to be `@generated`

Reply:
whoops, that was copy-and-paste typo, sorry.